host.json

```diff
@@ -8,6 +8,11 @@
       }
     }
   },
+  "extensions": {
+    "http": {
+      "routePrefix": ""
```

Comment:
This allows us to remove the /api in #3  I hope @emsesc  that is ok?

Reply:
@blueboxes Yes, works great!